src/cmap/conn/mod.rs

```diff
@@ -187,6 +193,11 @@ impl Connection {
         self.command_executing
     }
 
+    /// Checks if the connection experienced a network error and should be closed.
+    pub(super) fn is_errored(&self) -> bool {
```

Comment:
I'm not super fond of this name; what do you think of "has_errored"? I know that "is" is generally used for boolean predicates, but I think "has" is pretty clear that it's checking for a condition as well

Reply:
fixed